src/site.css

```diff
@@ -9,6 +9,18 @@
   --secondary: rgba(0, 0, 0, 0.1); /* Colours borders and modals */
   --text: #000; /* Colours most text */
   --header-text: #fff; /* Colours header text. Will fall back to `--text` if empty */
+
+  --badge-default-text: var(--text);
+  --badge-default-bg: var(--canvas);
+
+  --badge-warn-text: var(--text);
+  --badge-warn-bg: #dc3545;
```

Comment:
Apologies if I'm reading this wrong but should this be a variable defined in each of the themes instead? We may want to subtly change the shades or palette depending on what themes we use without causing everything to break on all the themes.

Reply:
So yes, but it's not required.

Each theme should absolutely specify their own colours for each type of badge, but since theming isn't really my strong suite I've opted to leave them as is.

Since the default styling is applied as `:root` rather than `body[theme=""]` these defaults will always be there, and in general the readability isn't that bad.

So below here's the current badges (without theme specific styling, with the following badge types: `warn`, `info`, `success`, `unkown`. In that order)

## Atom.io

![image](https://user-images.githubusercontent.com/26921489/226791261-dd64bd9e-83a1-4947-9df8-1182f62c96e9.png)

## GitHub Dark

![image](https://user-images.githubusercontent.com/26921489/226791290-f44542a4-937d-4581-922a-6bd57b4afb17.png)

## Dracula 

![image](https://user-images.githubusercontent.com/26921489/226791321-ce1649a4-53e3-49df-9e51-a1e23b086b1a.png)

## One Dark

![image](https://user-images.githubusercontent.com/26921489/226791351-6728eb41-5b4a-44cb-a680-151abec5969e.png)

## One Light

![image](https://user-images.githubusercontent.com/26921489/226791375-5495fe1a-62a7-4267-87d1-7df54abb039a.png)

---

So as you can see by just ensuring every theme has the default badge styling the rest are surprisingly readable. But if there's any frontend devs or people more design oriented than I am, it would absolutely be best to add some proper styling to these.

Do you think it's worth prioritizing on this PR? Or are they fine enough as is

Reply:
Honestly I think they are fine as they are now I can see them on each theme, I think I was just thinking that it may not be obvious that one is *meant* to theme them. Not a big deal either way.

Reply:
Hmm, you've got a point. I'll add a todo there before calling this one good then, just to try and make it obvious. We could even make an issue for it if we wanted to